### PR TITLE
chore: refactor to chain specific contracts

### PR DESCRIPTION
**Pull request by ajays97**
WIN-1511

### CODE REVIEW

**Comment by ajays97** [comment]
@sachushaji

**Review by sachushaji — APPROVED** [review]
LGTM, Thanks @ajays97

**Review by muhammadarifin88 — COMMENTED** [review]
Done

**Review by muhammadarifin88 — APPROVED** [review]

File: kava/WBTC.sol [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import "openzeppelin-solidity/contracts/token/ERC20/StandardToken.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/DetailedERC20.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/MintableToken.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/BurnableToken.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/PausableToken.sol";
+import "../evm-base-contracts/contracts/token/WrappedToken.sol";
+
+
+contract WBTC is WrappedToken, DetailedERC20("Wrapped BTC", "WBTC", 8) {}
```
Comment (Armanidashh) [inline kava/WBTC.sol:11]: 1

**Review by Armss9936 — APPROVED** [review]

**Review by firas9941 — APPROVED** [review]

**Pull request merged**